package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pr-calculator",
-  "version": "1.9.8",
+  "version": "1.10.0",
```

Comment:
I think this can just be a patch version as there are no new features.  Just a bug fix and a correction.  So `1.9.10`

`1.9.9` is already merged so you'll need to merge with main to avoid conflicts.